### DISCUSSION TITLE
Approach to contribution in Wagtail

### DISCUSSION THREAD

**Discussion by AN1KE7** [wagtail/gsoc#114] [General]
Hello, 
I am Aniket a CS bachelors student , I would appreciate some guidance on how I can effectively understand & analyze the Wagtail codebase and identify potential areas for contribution. Are there specific documentation resources, guidelines, or recommended tools that you would suggest for someone looking to delve into the codebase for the first time?
I look forward to your guidance and the opportunity to contribute to the Wagtail project.

Regards

**Comment by thibaudcolas** [comment]
Hi @AN1KE7, with a question like this I think it’d be better if you first shared what you’ve tried so far. There are a lot of docs that are very well sign-posted.

**Reply by AN1KE7** [reply]
Sir,
I have read the contributor's guide provided. I am familiar with python & beginner in open source with keen interest to understand your codebase, learn new things and contribute to it.
Also, I will be very happy to know if your organization is participating in GSOC 2024.

Warm regards

**Comment by iqraakhtar09** [comment]
Hi, I'm Iqra, and I'm a student of software engineering. I want to contribute to wagtail, and I would love to solve leetcode. I can work with Python, Django, Flask, Reactjs, and ML. I'm full of excitement to contribute to this organization. I want to know how to submit proposal here. Please let me know
